curriculum/challenges/english/25-front-end-development/quiz-css-pseudo-classes/66ed9002f45ce3ece4053eb6.md

```diff
@@ -17,439 +17,509 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What are pseudo-classes?
 
 #### --distractors--
 
-Placeholder distractor 1
+It is a CSS rule that creates additional content within an element when needed.
 
 ---
 
-Placeholder distractor 2
+It is a property that adjusts an element's size or layout on the page.
 
 ---
 
-Placeholder distractor 3
+They are selector used to add movement effects to an element during interactions.
 
 #### --answer--
 
-Placeholder answer
+They are keywords added to a selector that change how an element looks or behaves when it's in a specific state
+
+## --quiz--
```

Comment:
you should not add new headers, please go through the file and remove this header. there is only one per file at the top